routes/items.js

```diff
@@ -35,4 +35,15 @@ router.post( '/:id/completed', ( request, response ) => {
     .catch( error => response.json({ success: false, id, message: error.message }))
 })
 
+router.post( '/:id/edit', ( request, response ) => {
+  const Item = request.app.get( 'models' ).Item
+
+  const { id } = request.params
+  const { title, description } = request.body
```

Comment:
Is the validation within the scope of this issue? If so, what does that validation look like?